solutions/access-management/variables.tf

```diff
@@ -0,0 +1,1553 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key"
+  sensitive   = true
+}
+
+##############################################################################
+# Admin Observability Variables
+##############################################################################
+
+variable "admin_observability_ag_name" {
+  type        = string
+  description = "Name of the Administrator Observability access group"
+  default     = "admin-observability-group"
+}
+
+variable "admin_observability_ag_policies" {
+  type = map(object({
+    roles              = list(string)
+    account_management = optional(bool)
+    tags               = set(string)
+    resources = optional(list(object({
+      region               = optional(string)
+      attributes           = optional(map(string))
+      service              = optional(string)
+      resource_instance_id = optional(string)
+      resource_type        = optional(string)
+      resource             = optional(string)
+      resource_group_id    = optional(string)
+    })))
+    resource_attributes = optional(list(object({
+      name     = string
+      value    = string
+      operator = optional(string)
+    })))
+  }))
+  description = "A map of policies for the Administrator Observability access group, has a set of default policies that can be overridden"
+  default = {
+    "atracker" = {
+      roles = ["Administrator", "Writer"]
+      resources = [{
+        service = "atracker"
+      }]
+      tags = []
+    }
+    "observability" = {
```

Comment:
observability is logging and monitoring, however you have logdna below already, so should this be renamed monitoring?